### DISCUSSION TITLE
Sensors Filling up HDD

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#3714] [Unsupported Versions]
Hello SO community, 
So I have my sensor up and running but it appears that it fills the 1.5 tb of my 2 TB quickly, where as my manager node is not filling up its 8 TB drive. Am I wrong in my understanding of Log storage that Sensor forwards all Logs to the manager then the manager stores them or sends them to a search node. To this then I would only need small storage for my sensors and large for my manager or does that manager not store logs?

**Comment by darth-drew** [comment]
Your sensor node is likely filling up more with PCAP than logs, which is to be expected. PCAP is huge, with logs being a very small percentage of your local sensor storage, respectively. The PCAP stays resident on the sensor (forward node) as long as it can before being dropped on the floor to make room for newer PCAP. As far as logs, you are correct that they are forwarded to the manager, and eventually search nodes for storage. If you have dedicated search nodes though, the logs essentially flow through the manager, which acts as a middle man. Hopefully, this is helpful.

Check out this page for a better explanation: https://docs.securityonion.net/en/2.3/hardware.html
